nibabel/freesurfer/io.py

```diff
@@ -296,7 +326,20 @@ def write_morph_data(file_like, values, fnum=0):
 
 
 def read_annot(filepath, orig_ids=False):
-    """Read in a Freesurfer annotation from a .annot file.
+    """Read in a Freesurfer annotation from a `.annot` file.
+
+    An `.annot` file contains a sequence of vertices with a label (also known
+    as an "annotation value") associated with each vertex, and then a sequence
+    of colours corresponding to each label.
+
+    The colour table itself may be stored in either an "old-style" format, or
```

Comment:
AFAICT this PR will be the first introduction of "colour" to the docs as opposed to color. For the sake of consistency, do you mind switching to the American spelling?